File: backend/bus-api/src/main/java/com/busapp/buss_api/dto/RouteRequest.java
package com.busapp.buss_api.dto;

import lombok.*;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RouteRequest {
    private Integer departureProvinceId;
    private Integer destinationProvinceId;
    private Integer distanceKm;
    private Float durationHours;
    private Boolean isPopular;
}

File: android-admin-files/model/RouteRequest.kt
package com.example.busadmin.model

data class RouteRequest(
    val departureProvinceId: Int,
    val destinationProvinceId: Int,
    val distanceKm: Int? = null,
    val durationHours: Float? = null,
    val isPopular: Boolean? = null
)
